DNN Platform/Library/Entities/Users/UserEventArgs.cs

```diff
@@ -26,6 +26,6 @@ namespace DotNetNuke.Entities.Users
     public class UserEventArgs : EventArgs
     {
         public UserInfo User { get; set; }
-        public bool SendNotification { get; set; }
+        public bool SendEmailNotification { get; set; }
```

Comment:
No. By default it is false. I'm going to enable it in admin experience when `CreateUser` is invoked on a new user registration. So, email will be sent to admin and registered user.
Also, since we moved this to core part, this flag becomes available to other 3rd part applications and api consumers, so they can use overloaded `CreateUser` method to enable notifications as well.

Reply:
If UserEventArgs is part of the API, renaming a property would be a breaking change